### PR TITLE
add automation scenarios to test forward logs with tags

### PR DESCRIPTION
**Pull request by QiaolingTang**
Cases: https://url.corp.redhat.com/1a12958
Runner-v3 logs: https://url.corp.redhat.com/d7afc62

@anpingli PTAL, thanks.

### CODE REVIEW

**Review by anpingli — COMMENTED** [review]
/lgtm

File: lib/openshift/cluster_log_forwarder.rb [hunk 1]
```diff
@@ -60,6 +60,17 @@ def status_raw(user: nil, quiet: false, cached: true)
         return raw_resource(user: user, cached: cached, quiet: quiet).dig('status')
     end
 
+    def outputs_status(user: nil, quiet: false, cached: false)
```
Comment (liangxia) [inline lib/openshift/cluster_log_forwarder.rb:63]: We should default cached to true. Set cached to false in your scenario when you do not want to use cached value.

Comment (QiaolingTang) [reply lib/openshift/cluster_log_forwarder.rb]: Done, https://github.com/openshift/cucushift/pull/8173 is updated too.

**Pull request merged**